demos/slider.scss

```diff
@@ -17,3 +17,14 @@
 @import "./common";
 @import "../packages/mdc-slider/mdc-slider";
 @import "../packages/mdc-typography/mdc-typography";
+@import "../packages/mdc-theme/color-palette";
+
+.demo-slider--custom {
+  $main-color: $material-color-red-700;
+  $tick-color: $material-color-red-300;
```

Comment:
Is this supposed to be used?

Reply:
Good catch! Removed. I originally had a separate mixin for tick mark color, but then I removed it because I wasn't sure if it was supposed to be customizable. I figured we can always add that mixin back if necessary.